src/diffusers/pipelines/animatediff/pipeline_animatediff_video2video.py

```diff
@@ -662,12 +662,6 @@ def prepare_latents(
                 self.vae.to(dtype=torch.float32)
 
             if isinstance(generator, list):
-                if len(generator) != batch_size:
```

Comment:
oh where is it checked?

Reply:
Just a few lines above

![image](https://github.com/user-attachments/assets/12e9026c-4980-491f-96e9-816312c26bf6)